store/sqlstore/link_metadata_store.go

```diff
@@ -54,35 +54,31 @@ func (s SqlLinkMetadataStore) Save(metadata *model.LinkMetadata) store.StoreChan
 	})
 }
 
-func (s SqlLinkMetadataStore) Get(url string, timestamp int64) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var metadata *model.LinkMetadata
+func (s SqlLinkMetadataStore) Get(url string, timestamp int64) (*model.LinkMetadata, *model.AppError) {
+	var metadata *model.LinkMetadata
 
-		err := s.GetReplica().SelectOne(&metadata,
-			`SELECT
+	err := s.GetReplica().SelectOne(&metadata,
+		`SELECT
 				*
```

Comment:
Please keep here consistent the indentation (you are unindenting the first line of the string, but not the rest.